src/app/georeferencer/qgsgeorefdatapoint.cpp

```diff
@@ -77,6 +82,26 @@ void QgsGeorefDataPoint::setMapCoords( const QgsPointXY &p )
   }
 }
 
+void QgsGeorefDataPoint::setTransCoords( const QgsPointXY &p )
+{
+  mTransCoords = p;
+  if ( mGCPSourceItem )
+  {
+    mGCPSourceItem->update();
+  }
+  if ( mGCPDestinationItem )
+  {
+    mGCPDestinationItem->update();
+  }
+}
+
+QgsPointXY QgsGeorefDataPoint::transCoords() const
+{
+  if ( mTransCoords!=mMapCoords )
```

Comment:
Try to set up your pre commit git hook, it would have revealed a formatting issue here. 

Code-wise, why the if here? Simply return mTransCoords.

Reply:
TransCoords is not always set, I wasn't sure how to check an empty/unset qgspointxy as I wrote this.

Reply:
return mTransCoords.isEmpty() ? mMapCoords : mTransCoords ?